src/promptflow/promptflow/_cli/_pf/_run.py

```diff
@@ -601,5 +627,12 @@ def create_run(create_func: Callable, args):
     print(json.dumps(run._to_dict(), indent=4))
 
 
+@exception_handler("Delete run")
+def delete_run(names: str) -> None:
+    run_names = [name.strip() for name in names.split(",")]
+    pf_client = PFClient()
+    pf_client.runs.delete(runs=run_names)
```

Comment:
ask user for confirmation before actually does delete

Reply:
Okay